src/components/Preloader/Preloader.jsx

```diff
@@ -0,0 +1,11 @@
+import "./Preloader.css";
+
+function Preloader() {
+  return (
+    <>
```

Comment:
* `<>` empty tags are not needed if you have a common tag that wraps the whole markup of the component


```html

function Preloader() {
  return <div className="circle-preloader" />;
}
```